src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1287,6 +1287,22 @@ The following important rules apply to global variables and generic information:
             </genericInformation>
 ----
 
+=== Manage job labels
+
+A label is a property that can be attached to a job.
+The advantage of having labels is given by the possibility to group and manipulate a certain number of jobs by filtering based on this "custom status" or label.
+
```

Comment:
For example, a company can require that jobs which are having errors being reviewed by operators and mark them as "reviewed" when completed. Job labels can be used for this purpose.